scheduler/src/main/java/org/cloudfoundry/autoscaler/scheduler/quartz/AppScalingScheduleJob.java

```diff
@@ -0,0 +1,40 @@
+package org.cloudfoundry.autoscaler.scheduler.quartz;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.quartz.JobDataMap;
+import org.quartz.JobExecutionContext;
+import org.quartz.JobExecutionException;
+import org.quartz.JobKey;
+import org.springframework.scheduling.quartz.QuartzJobBean;
+import org.springframework.stereotype.Component;
+import org.springframework.web.context.support.SpringBeanAutowiringSupport;
+
+/**
+ * @author Fujitsu
+ *
+ */
+@Component
+public class AppScalingScheduleJob extends QuartzJobBean {
+	private Log logger = LogFactory.getLog(this.getClass());
+
+	@Override
+	protected void executeInternal(JobExecutionContext context) throws JobExecutionException {
+
+		SpringBeanAutowiringSupport.processInjectionBasedOnCurrentContext(this);
+		logger.info("==========================================");
```

Comment:
slf4j is just an abstraction layer not is not a logging implementation.so has to give the flexibility to use any loggng implementation  log4j or java logging. If you want to use log4j directly thats fine. We will go with that.